Marlin/example_configurations/delta/kossel_xl/Configuration.h

```diff
@@ -443,7 +443,7 @@
 
   // Center-to-center distance of the holes in the diagonal push rods.
   #define DELTA_DIAGONAL_ROD 317.3 + 2.5 // mm
-
+/*
```

Comment:
Don't mess with other configurations in this PR, unless related to the feature being modified. Certainly don't leave a block comment here like this. Pay attention to detail!